### ISSUE TITLE
include location shapefiles and link to locations.csv table

### ISSUE THREAD

**Issue by jhpoelen** [GoMexSI/interaction-data#9]
Currently, polygons describing location of study areas are manually copied from esri shapefile into the locations.csv files. Then, the centroid of this polygon is manually calculated and entered into the locations.csv also. 

Suggest to include the sources of the shapefiles and ensure that we can associate them to a location by some id or references. This way, the polygon and centroids can be inferred from the source shapefile. Also, the shapefile can be easily changed using standard esri mapping tools. Another advantage is that the shapefiles are accessible to others and version controlled.

**Comment by jsimons9** [comment]
We have been discussing this with you and I contacted John Froeschke as well since he uses shape files and R quite a bit.  Anyhow, after discussions with our GIS folks, I have discovered that each reference has three shape files; one for the centroid)s), one for the polygon(s), and one for the stations (Remember that each shape file actually consists of about 6 files, each with a different extension but the same name).  So the plan is to introduce a pointer in the reference spreadsheet to each of those three file types (centroid, polygon, stations). In addition, each of the predators also has three location shape files as above, so we will do the same for them.  We are going to start with the references, as they are easier to cope with. We plan to use the name (without any extension) as the pointer.  We can show you what this will look like, and send examples of the shape files.  I am assuming that you will have a plan as to where/how to store the shape files and how to access them.
